### PR TITLE
chore(c_sharp): highlight tweaks

### PR DESCRIPTION
**Pull request by amaanq**

### CODE REVIEW

File: queries/c_sharp/highlights.scm [hunk 1]
```diff
@@ -4,6 +4,9 @@
   (#eq? @keyword "value")
   (#has-ancestor? @keyword accessor_declaration))
 
+((identifier) @constant
+  (#eq? @constant "_"))
```
Comment (lucario387) [inline queries/c_sharp/highlights.scm:8]: Shouldn't this only be constants in some cases :thinking: 

I don't think all cases are fine, even though no normal person will use it as a variable

Comment (amaanq) [reply queries/c_sharp/highlights.scm:8]: Nah it's like that everywhere, it's a special variable in c#

Comment (lucario387) [reply queries/c_sharp/highlights.scm:8]: but that doesn't necessarily mean it's a constant variable :thinking: 

Im just skimming through C# docs: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards

> _ is also a valid identifier. When used outside of a supported context, _ is treated not as a discard but as a valid variable. If an identifier named _ is already in scope, the use of _ as a standalone discard can result in:
> - Accidental modification of the value of the in-scope _ variable by assigning it the value of the intended discard. For example:

Comment (amaanq) [reply queries/c_sharp/highlights.scm:8]: sure, I'm open to something else like `@variable.builtin`

Comment (lucario387) [reply queries/c_sharp/highlights.scm:8]: I'd be more happy with that.

**Review by lucario387 — APPROVED** [review]

**Pull request merged**